packages/apps/human-app/frontend/src/smart-contracts/chains.ts

```diff
@@ -53,3 +124,9 @@ export const chainsWithSCAddresses: ChainWithAddresses[] =
 export const chains: Chain[] = (
```

Comment:
Looks like this `chains` is now used only by `getNetworkName` utility, so let's give it a better name

Reply:
This seems to be not needed anymore after this CR refactor